lib/rules/no-get.js

```diff
@@ -2,6 +2,7 @@
 
 const types = require('../utils/types');
 const emberUtils = require('../utils/ember');
+const utils = require('../utils/utils');
```

Comment:
Could you do `const { isInLeftSideOfAssignmentExpression } =`?

Reply:
I could. We have a mix of both styles. Remind me if there's a specific benefit to that?

Reply:
It makes it easier to tell which functions it needs here, and `utils` isn't a great name for an object to have around.